hplsql/src/main/java/org/apache/hive/hplsql/Conn.java

```diff
@@ -146,15 +146,11 @@ synchronized Connection getConnection(String connName) throws Exception {
    * @throws Exception 
    */
   Connection openConnection(String connStr) throws Exception {
-    String driver = "org.apache.hadoop.hive.jdbc.HiveDriver";
     StringBuilder url = new StringBuilder();
     String usr = "";
     String pwd = "";
     if (connStr != null) {
       String[] c = connStr.split(";");
-      if (c.length >= 1) {
-        driver = c[0];
```

Comment:
if driver name is not expected in the `connStr` then `connStr.split(";");` need to be adjusted accordingly.